gTea/app/src/main/java/com/nechaieva/gtea/MenuFragment.java

```diff
@@ -14,6 +14,13 @@
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
+import com.nechaieva.gtea.DeepLink;
```

Comment:
Are these changes due to the Java version change?

Consider being consistent with the commit description/name